src/Database.js

```diff
@@ -167,8 +167,8 @@ class Database {
             reject(new Error(message));
           } else {
             /* eslint-disable no-param-reassign */
-            migration.up = up.replace(/^--.*?$/gm, '').trim();     // Remove comments
-            migration.down = down.replace(/^--.*?$/gm, '').trim(); // and trim whitespaces
+            migration.up = up.replace(/^-- .*?$/gm, '').trim();// Remove comments
```

Comment:
Thanks @theogravity 😁 

Quick quesiton: do we really need to remove comments? Simply doing `migration.up = up.trim()` (same as migration.down) would prevent future issues like the one I just had with TLS certificates ;)